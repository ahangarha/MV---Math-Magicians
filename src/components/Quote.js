import React from 'react';
import './Quote.css';

export default function Quote() {
  return (
    <blockquote>
      Mathematics is the most beautiful and most powerful creation of the human spirit.
      <span className="quote-ref">
        â€” Stefan Banach, Polish mathematician
      </span>
    </blockquote>
  );
}
