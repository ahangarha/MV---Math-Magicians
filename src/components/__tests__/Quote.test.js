import React from 'react';
import Renderer from 'react-test-renderer';
import Quote from '../Quote';

it('Renders correctly', () => {
  const tree = Renderer
    .create(<Quote />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
