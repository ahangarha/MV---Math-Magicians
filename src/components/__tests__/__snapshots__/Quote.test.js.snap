// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<blockquote>
  Mathematics is the most beautiful and most powerful creation of the human spirit.
  <span
    className="quote-ref"
  >
    â€” Stefan Banach, Polish mathematician
  </span>
</blockquote>
`;
